src/js/structs/NodesList.js

```diff
@@ -27,7 +25,7 @@ module.exports = class NodesList extends List {
 
       // Marathon API
       if (filters.ids) {
-        hosts = _.filter(hosts, function (host) {
+        hosts = hosts.filter(function (host) {
           return this.ids.indexOf(host.id) !== -1;
         }, {ids: filters.ids});
```

Comment:
hmm, like the last time this doesn't look like it needs the context. I'm going to remove it